.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 repos:
 - repo: https://github.com/adrienverge/yamllint.git
-  rev: v1.26.0
```

Comment:
@davisusanibar any specific reason why this upgrade is included in this PR?

Reply:
My problem with commitlint-pre-commit-hook was resolved by updating to the last version and applying the same update to yamlint to keep the version updated.